### PR TITLE
Add Plate 4 Images

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
In this PR, I edit the documentation and code to add Plate 4 into the download module to load from figshare.

### CODE REVIEW

File: 0.download_data/README.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ The plates for this project are located at [NF1 Schwann Cell Genotype Cell Paint
...
+
 To download the images for each plate and separate the metadata from the images, run the [download_plates.ipynb](download_plates.ipynb) notebook as a python script using the code block below:
 
 ```bash
 # Run this script in terminal
 cd 0.download_data
 source download_plates.sh
 ```
+
+To download 4 plates from figshare, it took about 50 minutes. There is the option to parallelize this in the future depending on needs.
```
Comment (gwaybio) [inline 0.download_data/README.md]: One sentence per line

File: README.md [hunk 2]
```diff
@@ -53,12 +53,27 @@ As well, the plates have different culturing conditions, where plate 3 cells wer
...
+
+![plate4_nf1_treat_dose.png](example_figures/plate4_nf1_platemap_treat_dose.png)
+
+There are three different siRNA constructs used in this plate, all with the same dose curve from 0.001 nM - 0.1 nM.
+Any well with a 0 nM concentration are not treated with a construct.
 
 ## Goal
 
-The goal of this project is to predict NF1 genotype from Schwann cell morphology. 
+It is important to study Schwann cells from NF1 patients because NF1 causes patients to develop neurofibromas, which are red bumps on the skin (tumors) that appear due to the loss of Ras-GAP neurofibromin.
```
Comment (gwaybio) [inline README.md]: There's a couple things to correct in this. It's probably worth looking up why NF1 deficient Schwann cells are important and the manifestations they cause (it's not red bumps)

**Review by gwaybio — APPROVED** [review]
Great work! I have two comments. Please address prior to merging

File: 0.download_data/README.md (hunk 1 above)
Comment (jenna-tomkinson) [reply 0.download_data/README.md]: :+1:

File: README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply README.md]: Sounds good! This statement was created in my first few months, so I would be happy to correct this with my new knowledge!

Comment (jenna-tomkinson) [reply README.md]: This statement has been corrected.

**Comment by jenna-tomkinson** [comment]
@gwaybio Thank you for the review! This PR has been merged.

**Pull request merged**